style.css

```diff
@@ -380,3 +380,339 @@
     top: 87.09%;
   }
 }
+
+.work {
+  background-color: rgb(229 229 229);
+  padding: 120px 55px;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+}
+
+.work-title1 {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin-bottom: 30px;
+}
+
+.work-head {
+  width: 327px;
+  height: 83px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande", "Lucida Sans", Arial, sans-serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #1c1a19;
+  display: flex;
+  align-items: center;
+  padding-bottom: 10px;
+  border-bottom: 1px solid black;
+  margin-bottom: 50px;
+}
+
+.work-head2 {
+  width: 327px;
+  height: 36px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande", "Lucida Sans", Arial, sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+}
+
+.work1-para {
+  width: 327px;
+  height: 147px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.work1-list {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.p {
+  margin-left: -60px;
+}
+
+.work1-list li {
+  list-style: none;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 20px;
+  color: #091e42;
+  padding: 8px;
+  border: 1px solid #8993a4;
+}
+
+.btn {
+  text-decoration: none;
+  color: white;
+  padding: 10px 8px;
+  background-color: rgb(255 107 0);
+  width: 124px;
+  text-align: center;
+}
+
+.one:hover {
+  color: rgb(245 88 0);
+  background: rgb(255 245 225);
+  border: 1px solid rgb(245 88 0);
+}
+
+.one:active {
+  background-color: rgb(224 83 4);
+  color: rgb(244 241 240);
+}
+
+.one:disabled {
+  background: #f4f1f0;
+  color: #5b5957;
+}
+
+.one:enabled {
+  color: white;
+  background-color: rgb(255 107 0);
+}
+
+.work-title2 {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin-bottom: 10px;
+  background: linear-gradient(180.45deg, rgb(38 38 38 / 0%) 0.75%, rgb(38 38 38 / 90%) 61.94%), url("img/img2.png");
+  padding-top: 75px;
+  padding-bottom: 0;
+}
+
+.work-head22 {
+  width: 295px;
+  height: 81px;
+  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande", "Lucida Sans", Arial, sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.work2-para {
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.ph {
+  margin-left: -73px;
+}
+
+.work2-list {
+  margin-top: -15px;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.work2-list li {
+  align-items: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 500;
```

Comment:
- [ ] A Very good design so far, you did a great job here but we can improve something to make it attractive to recruiters.

Here you can see how the width of the page is fixed but it should have a `width:100%` to re-flow according to the screen width.

Kindly fix this :point_down: 

![376](https://user-images.githubusercontent.com/88894525/205065591-98b2596c-58ed-49e4-bcff-588610f48d04.png)